### PR TITLE
docs: update mobile web page

### PR DESCRIPTION
**Pull request by KazuCocoa**
## Proposed changes

Got feedback about this page.

## Types of changes

What types of changes does your code introduce to Appium?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation Update (if none of the other choices apply)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I have read the [CONTRIBUTING](https://github.com/appium/appium/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla.js.foundation/appium/appium)
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: docs/en/writing-running-appium/web/mobile-web.md [hunk 1]
```diff
@@ -21,19 +21,21 @@ Then, use desired capabilities like these to run your test in mobile Safari:
 // javascript
 {
   platformName: 'iOS'
-  , platformVersion: '11.10'
+  , platformVersion: '13.2'
+  , automationName: 'XCUITest'
   , browserName: 'Safari'
-  , deviceName: 'iPhone 8'
+  , deviceName: 'iPhone '
```
Comment (mykola-mokhnach) [inline docs/en/writing-running-appium/web/mobile-web.md]: is such device names correct?

File: docs/en/writing-running-appium/web/mobile-web.md [hunk 2]
```diff
@@ -77,6 +82,13 @@ preferences so that the remote debugger port is open.
 
 ### Mobile Safari on a Real iOS Device
 
+#### For `XCUITest`
+
+We use [appium-ios-device](https://github.com/appium/appium-ios-device) to handle Safari.
```
Comment (mykola-mokhnach) [inline docs/en/writing-running-appium/web/mobile-web.md]: ... since Appium 1.15

File: docs/en/writing-running-appium/web/mobile-web.md [hunk 3]
```diff
@@ -248,6 +269,13 @@ automate the built-in browser. On all devices you can use the 'Chromium'
 
 #### Troubleshooting Chromedriver
 
+If your test target requires newer chrome driver version,
```
Comment (mykola-mokhnach) [inline docs/en/writing-running-appium/web/mobile-web.md]: I think `Chromedriver` should be spelled equally everywhere

File: docs/en/writing-running-appium/web/mobile-web.md [hunk 4]
```diff
@@ -171,7 +171,7 @@ class ContextTests extends PHPUnit_Extensions_AppiumTestCase
                 'platformVersion' => '7.1',
                 'automationName' => 'XCUITest',
                 'browserName' => 'Safari',
-                'deviceName' => 'iPhone Simulator'
+                'deviceName' => 'iPhone 11Simulator'
```
Comment (mykola-mokhnach) [inline docs/en/writing-running-appium/web/mobile-web.md]: typo here?

Comment (KazuCocoa) [reply docs/en/writing-running-appium/web/mobile-web.md]: oops....

**Review by imurchie — APPROVED** [review]

**Pull request merged**